### PR TITLE
Added integration document for cloudanix

### PR DESCRIPTION
**Pull request by ajinkya1986**
Added Cloudanix integration Document

### CODE REVIEW

File: content/docs/06-integrations/00-cloudanix.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: 'Cloudanix'
+metaTitle: 'cloudanix'
+metaDescription: 'Cloudanix container security helm chart for Spectro Cloud Palette'
```
Comment (taurusismysign) [inline content/docs/06-integrations/00-cloudanix.md:4]: Spectro CLoud are not two words? Please check

File: content/docs/06-integrations/00-cloudanix.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Cloudanix
+
+Cloudanix Container Security helm chart leverages Falco for to detect anamolous behaviours on kubernetes cluster and provides an interactive interface in Cloudanix Dashboard to view on which resources like pods, containers and nodes the activiy is occuring in real-time.
```
Comment (taurusismysign) [inline content/docs/06-integrations/00-cloudanix.md]: for and to. grammar.

Comment (taurusismysign) [reply content/docs/06-integrations/00-cloudanix.md]: @mpurusottamc Shall we mention Falco here?

File: content/docs/06-integrations/00-cloudanix.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+This Helm chart helps to install multiple Cloudanix services for container protection. The following services are installed
```
Comment (taurusismysign) [inline content/docs/06-integrations/00-cloudanix.md]: This helm chart installs 3 Cloudanix services to enable container security capabilities. The services are listed below:

File: content/docs/06-integrations/00-cloudanix.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+- inventory-service
+- threat-service
+- config-cron
+
+## Falco Chart
+
+The chart installs the Falco community chart which is a dependency in Chart.yaml.
```
Comment (taurusismysign) [inline content/docs/06-integrations/00-cloudanix.md]: grammar?

File: content/docs/06-integrations/00-cloudanix.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+# Versions Supported
+
+The following Cloudanix Container Security Helm Chart versions are supported to work with Palette.
```
Comment (taurusismysign) [inline content/docs/06-integrations/00-cloudanix.md]: Grammar. 

Cloudanix helm chart supports the following versions with Palette.

File: content/docs/06-integrations/00-cloudanix.md (hunk 4 above)
Comment (taurusismysign) [reply content/docs/06-integrations/00-cloudanix.md]: Also, there are multiple references to Falco .. user might get confused. We could remove it.

File: content/docs/06-integrations/00-cloudanix.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+<br />
+
+<Tabs>
+<Tabs.TabPane tab="0.0.3" key="0.0.3">
+
+* **0.0.3**
+
+<br />
+
+The following Falco Charts Version is supported to work with Cloudanix Charts.
```
Comment (taurusismysign) [inline content/docs/06-integrations/00-cloudanix.md]: remove Falco reference.

File: content/docs/06-integrations/00-cloudanix.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+<br />
+
+# Notable Parameters
+
+| Name | Supported Values | Default value | Description |
+| --- | --- | --- | --- |
+| authToken | provided by Cloudanix | NA | Authorization token is required for Cloudanix helm charts. |
+| accountId | provided by Cloudanix | NA | AccountID created by Cloudanix for the customer |
+| clusterName | Customer cluster name | NA | The name of the cluster on which the Cloudanix Container Securities Worloads should run. |
```
Comment (taurusismysign) [inline content/docs/06-integrations/00-cloudanix.md]: The name of the cluster on which Cloudanix container security capability will run.

File: content/docs/06-integrations/00-cloudanix.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+| falco.ebpf.enabled | `true`, `false` | NA | If installed with ebpf probe enabled the value should be `true`. |
+
+## Prerequisites
+
+- Minumum CPU Cores: 1
+- Minimum Memory Required: 20 MiB
+- The minimum kubernetes version supported for falco is 1.15
```
Comment (taurusismysign) [inline content/docs/06-integrations/00-cloudanix.md]: remove "for falco"

File: content/docs/06-integrations/00-cloudanix.md (hunk 2 above)
Comment (taurusismysign) [reply content/docs/06-integrations/00-cloudanix.md]: Cloudanix container security helm chart

File: content/docs/06-integrations/00-cloudanix.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Cloudanix
+
+Cloudanix Container Security helm chart leverages Falco for to detect anamolous behaviours on kubernetes cluster and provides an interactive interface in Cloudanix Dashboard to view on which resources like pods, containers and nodes the activiy is occuring in real-time.
+
+This Helm chart helps to install multiple Cloudanix services for container protection. The following services are installed
+
+- inventory-service
```
Comment (taurusismysign) [inline content/docs/06-integrations/00-cloudanix.md:23]: if we dont need it, pls remove.

File: content/docs/06-integrations/00-cloudanix.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+<br />
+
+# Notable Parameters
+
+| Name | Supported Values | Default value | Description |
+| --- | --- | --- | --- |
+| authToken | provided by Cloudanix | NA | Authorization token is required for Cloudanix helm charts. |
+| accountId | provided by Cloudanix | NA | AccountID created by Cloudanix for the customer |
+| clusterName | Customer cluster name | NA | The name of the cluster on which the Cloudanix Container Securities Worloads should run. |
+| falco.ebpf.enabled | `true`, `false` | NA | If installed with ebpf probe enabled the value should be `true`. |
```
Comment (taurusismysign) [inline content/docs/06-integrations/00-cloudanix.md]: what's the default value?

File: content/docs/06-integrations/00-cloudanix.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+<br />
+
+<Tabs>
+<Tabs.TabPane tab="0.32.0" key="0.32.0">
+
+* **0.32.0**
+
+<br />
+
+# Notable Parameters
```
Comment (taurusismysign) [inline content/docs/06-integrations/00-cloudanix.md]: @mpurusottamc I had a word with Ajinkya on this. My understanding is user dont have to provide any of these params and this is handled by SC for us. So, in that case we can remove it, right?

**Review by taurusismysign — CHANGES_REQUESTED** [review]
Pls make changes

File: content/docs/06-integrations/00-cloudanix.md (hunk 1 above)
Comment (ajinkya1986) [reply content/docs/06-integrations/00-cloudanix.md:4]: Two words used in other docs as well

Comment (mpurusottamc) [reply content/docs/06-integrations/00-cloudanix.md:4]: I think, two words is okay. I see that in the docs page as well.

File: content/docs/06-integrations/00-cloudanix.md (hunk 2 above)
Comment (mpurusottamc) [reply content/docs/06-integrations/00-cloudanix.md]: We should not.

File: content/docs/06-integrations/00-cloudanix.md (hunk 4 above)
Comment (mpurusottamc) [reply content/docs/06-integrations/00-cloudanix.md]: yes, let's remove any mention of falco.

File: content/docs/06-integrations/00-cloudanix.md (hunk 11 above)
Comment (mpurusottamc) [reply content/docs/06-integrations/00-cloudanix.md]: Yeah. All of these values are pre-populated usign macros. I guess, we can remove them. Let me check another add-on for reference.

File: content/docs/06-integrations/00-cloudanix.md [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Cloudanix
+
+Cloudanix container security helm chart detects anamolous behaviours on kubernetes cluster and provides an interactive interface in Cloudanix dashboard to view on which resources like pods, containers and nodes the activiy is occuring in real-time.
```
Comment (mpurusottamc) [inline content/docs/06-integrations/00-cloudanix.md]: We should also highlight that our helm chart detects threats. something like, `detects threats and ano.....`

Comment (mpurusottamc) [inline content/docs/06-integrations/00-cloudanix.md]: Spelling mistake, it should be `behaviors`.

Comment (mpurusottamc) [inline content/docs/06-integrations/00-cloudanix.md]: after occuring, we should have a comma. Also, spelling mistake. `occurring, in real-time`

File: content/docs/06-integrations/00-cloudanix.md [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+## Versions Supported
+
+Cloudanix helm chart supports the following versions with Palette.
+
+<br />
+
+<Tabs>
+<Tabs.TabPane tab="0.0.3" key="0.0.3">
+
+* **0.0.3**
```
Comment (mpurusottamc) [inline content/docs/06-integrations/00-cloudanix.md]: we should change version to 1.0.0

File: content/docs/06-integrations/00-cloudanix.md [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+
+<br />
+
+## Prerequisites
+
+- Minumum CPU Cores: 1
+- Minimum Memory Required: 20 MiB
+- The minimum kubernetes version supported is 1.15
```
Comment (mpurusottamc) [inline content/docs/06-integrations/00-cloudanix.md]: kubernetes, K should be upper case.

**Review by mpurusottamc — CHANGES_REQUESTED** [review]
Please make the changes. Rest looks good.

**Review by mpurusottamc — APPROVED** [review]
looks good. Thanks for making the changes.

**Pull request merged**